tests/integration/framework/process/sentry/sentry.go

```
@@ -245,7 +347,10 @@ func (s *Sentry) DialGRPC(t *testing.T, ctx context.Context, sentryID string) *g
 	)
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		require.NoError(t, conn.Close())
+		if err := conn.Close(); err != nil {
+			// Log the error but don't fail the test if connection is already closed
+			t.Logf("Warning: gRPC connection close error (may be already closed): %v", err)
+		}
```

Please can we error type match here, instead of ignoring the error.